### PR TITLE
Document the ROUTE message

### PR DESCRIPTION
**Pull request by robsdedude**

### CODE REVIEW

File: bolt/bolt-protocol-message-specification-4.md [hunk 1]
```diff
@@ -1363,7 +1377,94 @@ Example:
...
+#### Synopsis
+
+```
+ROUTE {routing} [bookmarks] "db"
+```
+
+Example 1:
+
 ```
+RUN {"address": "x.example.com:7687"} [] null
```
Comment (gjmwoods) [inline bolt/bolt-protocol-message-specification-4.md]:
```suggestion
ROUTE {"address": "x.example.com:7687"} [] null
```

File: bolt/bolt-protocol-message-specification-4.md [hunk 2]
```diff
@@ -1363,7 +1377,94 @@ Example:
...
+```
+
+Example 2:
+
+```
+RUN {"address": "x.example.com:9001", "policy": "example_policy_routing_context", "region": "example_region_routing_context"} ["neo4j-bookmark-transaction:1", "neo4j-bookmark-transaction:2"] "example_database"
```
Comment (gjmwoods) [inline bolt/bolt-protocol-message-specification-4.md]:
```suggestion
ROUTE {"address": "x.example.com:9001", "policy": "example_policy_routing_context", "region": "example_region_routing_context"} ["neo4j-bookmark-transaction:1", "neo4j-bookmark-transaction:2"] "example_database"
```

**Review by gjmwoods — CHANGES_REQUESTED** [review]
More great stuff just a couple of typos

File: bolt/bolt-protocol-message-specification-4.md [hunk 3]
```diff
@@ -1363,7 +1377,94 @@ Example:
 
 ```
 FAILURE {"code": "Example.Failure.Code", "message": "example failure"}
+
```
Comment (robsdedude) [inline bolt/bolt-protocol-message-specification-4.md]:
```suggestion
```

File: bolt/bolt-protocol-message-specification-4.md (hunk 1 above)
Comment (robsdedude) [reply bolt/bolt-protocol-message-specification-4.md]: Copy paste errors \o/

**Review by gjmwoods — APPROVED** [review]

**Pull request merged**